apps/birdbyt/birdbyt.star

```diff
@@ -0,0 +1,396 @@
+"""
+Applet: Birdbyt
+Summary: Show nearby bird sightings
+Description: Displays a random bird sighting near a specific location.
+Author: Becky Sweger
+"""
+
+load("cache.star", "cache")
```

Comment:
Wanted to mention that the http module now has built in caching. Just pass `ttl_seconds` to `http.get()`. Might be worth migrating over to that in a later PR. It'll simplify the code a bit, and also future proofs things a bit as the cache module may change a bit down the road.

Reply:
Sounds great--will to a follow-up PR after the app is published!